data/rETH/data.json

```diff
@@ -2,12 +2,17 @@
   "name": "Rocket Pool ETH",
   "symbol": "rETH",
   "decimals": 18,
+  "website": "https://rocketpool.net/",
+  "twitter": "@Rocket_Pool",
   "tokens": {
     "ethereum": {
       "address": "0xae78736cd615f374d3085123a210448e74fc6393"
     },
     "optimism": {
       "address": "0x9bcef72be871e61ed4fbbc7630889bee758eb81d"
+    },
+    "goerli": {
```

Comment:
is there a corresponding optimism goerli address for rETH? This will make the asset bridgeable between goerli and optimism goerli

Reply:
It now exists and the address has been added